app/src/main/java/org/sopt/melon/presentation/main/MainScreen.kt

```diff
@@ -1,12 +1,96 @@
 package org.sopt.melon.presentation.main
 
-import androidx.compose.material3.Text
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.EnterTransition
+import androidx.compose.animation.ExitTransition
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.animation.slideIn
+import androidx.compose.animation.slideOut
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.navigationBarsPadding
+import androidx.compose.material3.Scaffold
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.IntOffset
+import androidx.navigation.compose.NavHost
+import kotlinx.collections.immutable.toImmutableList
+import org.sopt.melon.presentation.drawer.drawerGraph
+import org.sopt.melon.presentation.foryou.navigation.forYouGraph
+import org.sopt.melon.presentation.home.navigation.homeGraph
+import org.sopt.melon.presentation.main.component.MainBottomBar
+import org.sopt.melon.presentation.mixup.navigation.mixUpGraph
+import org.sopt.melon.presentation.mixup.navigation.navigateToMixUp
+import org.sopt.melon.presentation.search.searchGraph
+import org.sopt.melon.presentation.shortcut.shortCutGraph
 
 @Composable
 fun MainScreen(
-    modifier: Modifier = Modifier,
+    navigator: MainNavigator = rememberMainNavigator(),
 ) {
-    Text("test")
+    Scaffold(
+        bottomBar = {
+            AnimatedVisibility(
+                visible = navigator.showBottomBar(),
+                enter = fadeIn() + slideIn { IntOffset(0, it.height) },
+                exit = fadeOut() + slideOut { IntOffset(0, it.height) },
+            ) {
+                Column(
+                    modifier =
+                        Modifier
+                            .navigationBarsPadding(),
+                ) {
+                    MainBottomBar(
+                        tabs = MainTab.entries.toImmutableList(),
+                        currentTab = navigator.currentTab,
+                        onTabSelected = navigator::navigate,
+                        modifier = Modifier,
+                    )
+                }
```

Comment:
navigationBarsPadding을 Column에 넣어준 이유가 궁금해요!
```suggestion
                    MainBottomBar(
                        tabs = MainTab.entries.toImmutableList(),
                        currentTab = navigator.currentTab,
                        onTabSelected = navigator::navigate,
                        modifier = Modifier.navigationBarsPadding(),
                    )
                }
```
이렇게 할 때와 차이가 궁금..

Reply:
아 이거 border 넣으려다가 수정을 안했네요 바텀바 안에다 넣게 되게 되어서 제안주신 방식으로 적용하는게 좋겠습니다 !